### PR TITLE
Remove khashmap and tstate

### PR DESCRIPTION
**Pull request by alexttx**
Signed-off-by: Alex Tomlinson <4172734+alexttx@users.noreply.github.com>

### CODE REVIEW

**Comment by tristan957** [comment]
-434. I like that quite a bit :)

File: lib/cn/cndb.c [hunk 1]
```diff
@@ -2194,6 +2194,7 @@ cndb_cn_count(struct cndb *cndb, u32 *cnt)
     return 0;
 }
 
+/* [HSE_REVISIT] tstate used to use cndb_cn_blob_get().  does anybody else?  can we lose it? */
```
Comment (tristan957) [inline lib/cn/cndb.c:2197]: Can we just do a grep and see if anybody else is calling this? What would be required to figure this out?

**Review by tristan957 — APPROVED** [review]

File: lib/cn/cndb.c (hunk 1 above)
Comment (alexttx) [reply lib/cn/cndb.c:2197]: I think we hand that off to @gsramdasi  since he's deep into CNDB at this point.

Comment (gsramdasi) [reply lib/cn/cndb.c:2197]: Yes, I've removed all references to this function

**Review by nabeelmmd — APPROVED** [review]

**Pull request merged**